recipe/meta.yaml

```diff
@@ -50,10 +45,8 @@ about:
   license: Apache-2.0 AND BSD-3-Clause
   license_file:
     - cugraph/LICENSE
-    - rmm/LICENSE
```

Comment:
This still needs to be here because `rmm` code is used in `libcugraph`

Reply:
No as `librmm` is now a `run` requirement via `run_exports`

https://github.com/conda-forge/librmm-feedstock/blob/577d2262120f2dd826728a59eef46682e877516e/recipe/meta.yaml#L16-L17

Reply:
`run_exports` for a header only library doesn't make sense. What should happen is that `run_exports` of `libcugraph` should have `librmm` in addition to `libcugraph`

Reply:
Agree it is unusual. Though the issue is `libcugraph` is exposing `librmm` objects in its API. So unfortunately it needs to be `run` dependency until that can be addressed

Reply:
No, it can make sense if the header-only library is to be used (compiled) at runtime. Thrust/CUB is a good example. Experience showed that if Thrust seen at build time is not the same as at runtime, strange data corruption happened.

Reply:
> No, it can make sense if the header-only library is to be used (compiled) at runtime. Thrust/CUB is a good example. Experience showed that if Thrust seen at build time is not the same as at runtime, strange data corruption happened.

That's why `run_constrained` is there.

> Though the issue is libcugraph is exposing librmm objects in its API. So unfortunately it needs to be run dependency until that can be addressed

That's why I said `libcugraph` needs `librmm` in its `run_exports`.

Reply:
TBH am not sure that whether it will be needed in `libcugraph`'s dependencies, of which we are really only thinking of one atm (`cupy`)

Anyways not trying to argue the situation is perfect here. We know it is a bit messy and discussed this internally as well. I think the hope is we can iron a lot of these things out in the subsequent release